### ISSUE TITLE
SVM in tensorflow

### ISSUE THREAD

**Issue by mbjoseph** [mbjoseph/tensorphloem#5]
It would be nice to have the SVM implemented in Tensorflow so that we can better scale up to larger data sets and GPU setups. There may be some useful material here to help with defining the loss function, etc.:

https://github.com/AidanGG/tensorflow_tmva/tree/master/tensorflow_tmva/classification/svm

https://github.com/eakbas/tf-svm
